shared/source/os_interface/windows/driver_info_windows.cpp

```diff
@@ -34,20 +34,30 @@ std::string getCurrentLibraryPath() {
 
 namespace NEO {
 
-DriverInfo *DriverInfo::create(const HardwareInfo *hwInfo, OSInterface *osInterface) {
-    if (osInterface) {
-        auto wddm = osInterface->get()->getWddm();
-        DEBUG_BREAK_IF(wddm == nullptr);
+DriverInfo *DriverInfo::create(const HardwareInfo *hwInfo, const OSInterface *osInterface) {
+    if (osInterface == nullptr) {
+        return nullptr;
+    }
+
+    auto osInterfaceImpl = osInterface->get();
+    if (osInterfaceImpl == nullptr) {
+        DEBUG_BREAK_IF(true);
+        return nullptr;
+    }
```

Comment:
No need for such check

Reply:
The check has been replaced with `UNRECOVERABLE_IF()`.

Reply:
drop the whole if statement and put UNRECOVERABLE_IF(osInterfaceImpl == nullptr)

Reply:
It is already done. Please see [shared/source/os_interface/windows/driver_info_windows.cpp:43](https://github.com/intel/compute-runtime/pull/374/files#diff-af2031d519389b27950d3dae2ce285d8d642137f6dfe176fe39331aad3fd697cR41)